Cargo.toml

```diff
@@ -26,11 +26,12 @@ tokio-condvar = { version="0.3", features=["parking_lot"] }
 tokio-uring = { version="0.5", optional=true }
 tokio-uring-executor = { package="kioto-uring-executor", version="0.1", optional=true }
 bloomfilter = { version="1", optional=true }
+rand = "0.8.5"
```

Comment:
I think you can move this to "dev-dependencies"

Reply:
Done. [679eae2](https://github.com/kaimast/lsm-rs/pull/38/commits/679eae253fea2d5aec721ca14ea68671975fdfb5)